onewire_bus/include/onewire_bus_impl_rmt.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2022-2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include <stdint.h>
+#include "esp_err.h"
+#include "onewire_types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief 1-Wire bus RMT specific configuration
+ */
+typedef struct {
+    uint32_t max_rx_bytes; /*!< Set the largest possible single receive size,
```

Comment:
Could we add a note that this is relevent only for esp32 and esp32s2?

Reply:
no, not only for esp32 and esp32s2. It decides the receive buffer we used to save the RMT symbols. But for esp32 and esp32s2, this configuration has more meaning, we also use it to set the proper size of the RMT hardware memory block.